curses/client.cpp

```diff
@@ -507,14 +502,14 @@ int main(int argc, char * argv[])
 				{
 					case '+':
 						dac_number++;
-						if (dac_number >= MAX_DACS)
+						if (dac_number >= number_of_dacs)
```

Comment:
The list doesn't exist but renaming dac_number to current_dac_index should help.

The contents of the big switch statement has been cleaned up.